refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,887 @@
+
+In that way, interoperability CAN always be expected at the level of [Terms](#terms).
+
+### Extending the vocabulary
+
+Systems MAY specify the vocabulary used to express data being exchanged. The value indicating this version of this vocabulary is `priv.1.0`.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `vocab` | 0-1 | The name of the vocabulary used to describe data. In absence of indication `priv.1.0` is assumed. |
+
+When [addition additional terms](#extending-terms) is not sufficient, and additional Concepts or Properties are needed, it is possible to extend the vocabulary.
+
+This vocabulary is extended by defining a new identifier for the new vocabulary.
+New vocabularies MAY include other vocabularies as long as their sets of concepts, properties and terms are disjoint.  
+
+## Examples
+
+A [JSON schema of the PRIV](./priv.schema.json) is provided for convenience. To illustrate a request, let us take the example of a data subject wanting to know if a an Organization (and any of its partners if applicable) have any data on them, and if so, have their contact data deleted.
+
+This request can be represented with the following json data:
+```
+{
+  "$schema":"https://blindnet.io/schemas/priv.schema.json",
+  "request-id": "8f9066c6-1c6c-42a0-9993-e88c98d0e84d",
+  "date": "2022-06-02T14:40:39+0000",
+  "data-subject":[{
+    "dsid-schema": "email-sha-256",
+    "dsid":"7cac89a56bbf998c996f33e0b2d3bad578e05f3af8d64793c0bcac46b8c260dc"
+  }],
+  "demands":[{
+    "demand-id":"496294eb-5293-47dd-aaf8-494a0cb09134",
+    "action":"TRANSPARENCY.KNOWN"
+  },{
+    "demand-id":"86bbb28a-eee6-45e6-81d6-7101de32374b",
+    "action":"DELETE",
+    "restrictions":[{
+      "data-categories":["CONTACT"]
+    }]
+  }],
+  "target":"PARTNERS"
+
+}
+
+```
+In [here](./examples.md) we provide an overview of various Privacy Requests that a Data Subject might want to formulate according to [supported legislation](#supported-legislation) and we give indications on how each of those can be modeled with the Privacy Request Interchange Vocabulary.
+
+## Design Implications for Systems Implementing PRIV
+
+### Remembering Transfers
+
+When data about Data Subjects is transmitted from one system to another, in order to be able to process the [Targets property](#targets), and in order to reply to `TRANSPARENCY.WHO` and `TRANSPARENCY.PROVENANCE` demands, Systems MUST keep track of:
+- ID of Systesm of origin and destination, in the format of URI according to [RFC3986 of IETF](https://www.rfc-editor.org/rfc/rfc3986)
+- Full Data Capture metadata objects
+- Consents (`consent-id`) associated to the data being transferred
+- Data Subject Identities (`dsid`,`dsid-schema`) pairs associated to the data being transferred
```

Comment:
This is part of the Data Capture metadata objects, right?

Reply:
Consents, can be generated after (see [operations over consents](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/expected-behavior.md#operations-over-consents)).

While some Data Subject Identities are indeed part of Data Capture metadata objects, a System receiving the data might associate to the data some additional Data Subject Identities that it know about (that were unknown to the System of data origin, and as such not part of the Data Capture).